idflakies-core/src/main/java/edu/illinois/cs/dt/tools/utility/Tuscan.java

```diff
@@ -0,0 +1,168 @@
+package edu.illinois.cs.dt.tools.utility;
+
+import scala.annotation.meta.getter;
+
+public class Tuscan {
+    public void main(int arg) {
```

Comment:
Maybe change this `main` method name to something more descriptive, like `generateTuscanSquare`.